src/linear_algebra/finsupp.lean

```diff
@@ -8,12 +8,13 @@ Linear structures on function with finite support `α →₀ M`.
 import data.finsupp linear_algebra.basic
 
 noncomputable theory
-local attribute [instance, priority 10] classical.prop_decidable
 
 open lattice set linear_map submodule
 
 namespace finsupp
 
+local attribute [instance, priority 10] classical.prop_decidable
```

Comment:
Maybe I'm missing something. Why don't you simply write `open_locale classical`?

Reply:
Because I just moved this from a few lines above. Changed.